[OS X 10.10 Yosemite] Installing ./xs failed.. The XS/C++ code failed to compile, aborting.

Just tried to build Slic3r on OS X 10.10(yosemite) and got the following (This is perl 5, version 18, subversion 5 (v5.18.2)):


App::cpanminus is up to date. (1.7014)
Class::XSAccessor is up to date. (1.19)
Encode::Locale is up to date. (1.03)
ExtUtils::MakeMaker is up to date. (7.00)
ExtUtils::ParseXS is up to date. (3.24)
skipping R/RJ/RJBS/perl-5.20.0.tar.gz
File::Spec is up to date. (3.47)
Getopt::Long is up to date. (2.42)
IO::Scalar is up to date. (2.110)
Math::PlanePath is up to date. (117)
Module::Build::WithXSpp is up to date. (0.14)
Moo is up to date. (1.006001)
POSIX is up to date. (1.32)
Scalar::Util is up to date. (1.41)
Test::Harness is up to date. (3.33)
Test::More is up to date. (1.001008)
Thread::Semaphore is up to date. (2.12)
Time::HiRes is up to date. (1.9726)
XML::SAX::ExpatXS is up to date. (1.33)
cpanm (App::cpanminus) 1.7014 on perl 5.018002 built for darwin-thread-multi-2level
Work directory is /Users/aminiis/.cpanm/work/1414195658.4149
You have make /usr/bin/make
You have LWP 6.05
You have /usr/bin/tar: bsdtar 2.8.3 - libarchive 2.8.3
You have /usr/bin/unzip
--> Working on ./xs
Entering /Users/aminiis/Slic3r/xs
Checking if you have Module::Build 0.36 ... Yes (0.4003)
Running Build.PL
Configuring /Users/aminiis/Slic3r/xs ... Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Slic3r-XS' version '0.01'
OK
Checking dependencies from MYMETA.json ...
Checking if you have ExtUtils::ParseXS 3.18 ... Yes (3.24)
Checking if you have ExtUtils::Typemaps::Default 1.05 ... Yes (1.05)
Checking if you have Module::Build 0.3601 ... Yes (0.4003)
Checking if you have Test::More 0 ... Yes (1.001008)
Checking if you have ExtUtils::XSpp 0.18 ... Yes (0.18)
Checking if you have ExtUtils::Typemap 1.00 ... Yes (1.00)
Building and testing Slic3r-XS-0.01 ... Building Slic3r-XS
Processing XS typemap files...
Generating main XS file...
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/MotionPlanner.o src/libslic3r/MotionPlanner.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Print.o src/libslic3r/Print.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Layer.o src/libslic3r/Layer.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/util.o src/admesh/util.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Polygon.o src/libslic3r/Polygon.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/poly2tri/sweep/sweep.o src/poly2tri/sweep/sweep.cc
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/poly2tri/common/shapes.o src/poly2tri/common/shapes.cc
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Config.o src/libslic3r/Config.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/poly2tri/sweep/cdt.o src/poly2tri/sweep/cdt.cc
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/PrintObject.o src/libslic3r/PrintObject.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/MultiPoint.o src/libslic3r/MultiPoint.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Line.o src/libslic3r/Line.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/ClipperUtils.o src/libslic3r/ClipperUtils.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/SurfaceCollection.o src/libslic3r/SurfaceCollection.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/shared.o src/admesh/shared.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/BoundingBox.o src/libslic3r/BoundingBox.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/normals.o src/admesh/normals.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/TriangleMesh.o src/libslic3r/TriangleMesh.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/LayerRegion.o src/libslic3r/LayerRegion.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Model.o src/libslic3r/Model.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/ExtrusionEntity.o src/libslic3r/ExtrusionEntity.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/PlaceholderParser.o src/libslic3r/PlaceholderParser.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/utils.o src/libslic3r/utils.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/PrintRegion.o src/libslic3r/PrintRegion.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/SVG.o src/libslic3r/SVG.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Polyline.o src/libslic3r/Polyline.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Flow.o src/libslic3r/Flow.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/poly2tri/sweep/advancing_front.o src/poly2tri/sweep/advancing_front.cc
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/stl_io.o src/admesh/stl_io.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/PolylineCollection.o src/libslic3r/PolylineCollection.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Surface.o src/libslic3r/Surface.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Extruder.o src/libslic3r/Extruder.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/PrintConfig.o src/libslic3r/PrintConfig.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Geometry.o src/libslic3r/Geometry.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/clipper.o src/clipper.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/poly2tri/sweep/sweep_context.o src/poly2tri/sweep/sweep_context.cc
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/connect.o src/admesh/connect.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/Point.o src/libslic3r/Point.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/ExPolygon.o src/libslic3r/ExPolygon.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/ExtrusionEntityCollection.o src/libslic3r/ExtrusionEntityCollection.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/admesh/stlinit.o src/admesh/stlinit.c
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/polypartition.o src/polypartition.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o src/libslic3r/ExPolygonCollection.o src/libslic3r/ExPolygonCollection.cpp
cc -I/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -DVERSION="0.01" -DXS_VERSION="0.01" -xc++ -D_GLIBCXX_USE_C99 -DHAS_BOOL -DNOGDI -DSLIC3RXS -Isrc -Ibuildtmp -c -arch x86_64 -arch i386 -g -pipe -fno-common -DPERL_DARWIN -fno-strict-aliasing -fstack-protector -Os -o buildtmp/XS.o buildtmp/XS.c
buildtmp/XS.c:28783:1: warning: duplicate 'extern' declaration specifier [-Wduplicate-decl-specifier]
XS_EXTERNAL(boot_Slic3r__XS); /* prototype to pass -Wmissing-prototypes */
^
/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE/XSUB.h:145:33: note: expanded from macro 'XS_EXTERNAL'
#      define XS_EXTERNAL(name) extern "C" XSPROTO(name)
                                ^
1 warning generated.
buildtmp/XS.c:28783:1: warning: duplicate 'extern' declaration specifier [-Wduplicate-decl-specifier]
XS_EXTERNAL(boot_Slic3r__XS); /* prototype to pass -Wmissing-prototypes */
^
/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE/XSUB.h:145:33: note: expanded from macro 'XS_EXTERNAL'
#      define XS_EXTERNAL(name) extern "C" XSPROTO(name)
                                ^
1 warning generated.
ExtUtils::Mkbootstrap::Mkbootstrap('blib/arch/auto/Slic3r/XS/XS.bs')
env LD_RUN_PATH=/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE cc -mmacosx-version-min=10.10 -arch x86_64 -arch i386 -bundle -undefined dynamic_lookup -fstack-protector -o blib/arch/auto/Slic3r/XS/XS.bundle buildtmp/XS.o src/libslic3r/MotionPlanner.o src/libslic3r/Print.o src/libslic3r/Layer.o src/admesh/util.o src/libslic3r/Polygon.o src/poly2tri/sweep/sweep.o src/poly2tri/common/shapes.o src/libslic3r/Config.o src/poly2tri/sweep/cdt.o src/libslic3r/PrintObject.o src/libslic3r/MultiPoint.o src/libslic3r/Line.o src/libslic3r/ClipperUtils.o src/libslic3r/SurfaceCollection.o src/admesh/shared.o src/libslic3r/BoundingBox.o src/admesh/normals.o src/libslic3r/TriangleMesh.o src/libslic3r/LayerRegion.o src/libslic3r/Model.o src/libslic3r/ExtrusionEntity.o src/libslic3r/PlaceholderParser.o src/libslic3r/utils.o src/libslic3r/PrintRegion.o src/libslic3r/SVG.o src/libslic3r/Polyline.o src/libslic3r/Flow.o src/poly2tri/sweep/advancing_front.o src/admesh/stl_io.o src/libslic3r/PolylineCollection.o src/libslic3r/Surface.o src/libslic3r/Extruder.o src/libslic3r/PrintConfig.o src/libslic3r/Geometry.o src/clipper.o src/poly2tri/sweep/sweep_context.o src/admesh/connect.o src/libslic3r/Point.o src/libslic3r/ExPolygon.o src/libslic3r/ExtrusionEntityCollection.o src/admesh/stlinit.o src/polypartition.o src/libslic3r/ExPolygonCollection.o -lstdc++
t/01_trianglemesh.t ............. ok     
t/03_point.t .................... ok     
t/04_expolygon.t ................ ok     
t/05_surface.t .................. ok     
t/06_polygon.t .................. ok     
t/07_extrusionpath.t ............ ok   
t/08_extrusionloop.t ............ ok     
t/09_polyline.t ................. ok     
t/10_line.t ..................... ok     
t/11_clipper.t .................. ok     
t/12_extrusionpathcollection.t .. ok     
t/13_polylinecollection.t ....... ok   
t/14_geometry.t ................. ok   
t/15_config.t ................... 1/105 
#   Failed test 'deserialize points'
#   at t/15_config.t line 65.
#     Structures begin differing at:
#          $got->[0][0] = '0'
#     $expected->[0][0] = '20'

#   Failed test 'deserialize points'
#   at t/15_config.t line 65.
#     Structures begin differing at:
#          $got->[0][0] = '0'
#     $expected->[0][0] = '20'

#   Failed test 'apply dynamic over dynamic'
#   at t/15_config.t line 169.
#     Structures begin differing at:
#          $got->[1][0] = '20'
#     $expected->[1][0] = '0'
# Looks like you failed 3 tests of 105.
t/15_config.t ................... Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/105 subtests 
t/16_flow.t ..................... ok   
t/17_boundingbox.t .............. ok   
t/18_motionplanner.t ............ ok     
t/19_model.t .................... ok   
t/20_print.t .................... ok   

Test Summary Report
-------------------
t/15_config.t                 (Wstat: 768 Tests: 105 Failed: 3)
  Failed tests:  23, 71, 101
  Non-zero exit status: 3
Files=19, Tests=423,  1 wallclock secs ( 0.10 usr  0.05 sys +  0.55 cusr  0.13 csys =  0.83 CPU)
Result: FAIL
Failed 1/19 test programs. 3/423 subtests failed.
FAIL
! Installing ./xs failed. See /Users/aminiis/.cpanm/work/1414195658.4149/build.log for details. Retry with --force to force install it.
The XS/C++ code failed to compile, aborting

Any help ?

--- a/xs/src/libslic3r/Config.hpp
+++ b/xs/src/libslic3r/Config.hpp
@@ -295,18 +295,21 @@ class ConfigOptionPoints : public ConfigOption, public ConfigOptionVector<Pointf
     };
     
     bool deserialize(std::string str) {
-        std::vector<Pointf> values;
+        this->values.clear();
         std::istringstream is(str);
         std::string point_str;
         while (std::getline(is, point_str, ',')) {
             Pointf point;
             std::istringstream iss(point_str);
-            iss >> point.x;
-            iss.ignore(std::numeric_limits<std::streamsize>::max(), 'x');
-            iss >> point.y;
-            values.push_back(point);
+            std::string coord_str;
+            if (std::getline(iss, coord_str, 'x')) {
+                std::istringstream(coord_str) >> point.x;
+                if (std::getline(iss, coord_str, 'x')) {
+                    std::istringstream(coord_str) >> point.y;
+                }
+            }
+            this->values.push_back(point);
         }
-        this->values = values;
         return true;
     };
 };

